GiniVision/Classes/Core/GiniScreenAPICoordinator.swift

```diff
@@ -153,17 +186,16 @@ extension GiniScreenAPICoordinator {
     }
     
     @objc func showAnalysisScreen() {
-        let documentToShow = visionDocuments[0]
-        if let didReview = visionDelegate?.didReview(document:withChanges:) {
-            didReview(documentToShow, changesOnReview)
-        } else if let didReview = visionDelegate?.didReview(_:withChanges:) {
-            didReview(documentToShow.data, changesOnReview)
+        if let didReviewDocuments = visionDelegate?.didReview(documents:) {
+            didReviewDocuments(visionDocuments)
+        } else if let didReview = visionDelegate?.didReview(document:withChanges:) {
+            didReview(visionDocuments[0], false)
```

Comment:
I know we already discussed this in the past, but are you sure this is fine? Even if you're sure it will never be empty, maybe it still makes sense to have an assert to be extra explicit?

Reply:
The thing is that I haven't decided yet if I want to make that method also unavailable and keep only the one that handle one or several documents at the same time. Of course looking for the first element is completely wrong. I will change it for this PR, but I will probably delete it in a future PR.